src/step_definitions/elements.ts

```diff
@@ -188,180 +93,35 @@ Then(/^there are "([^"]*)" dropdown list elements with following options:$/, fun
   });
 });
 
-Then(/^there is element "([^"]*)" with value "([^"]*)"$/, function(elementName, value) {
-  const pageElement = this.currentPage.getElement(elementName);
-
-  return this.currentPage.waitForVisibilityOf(elementName).then(() => {
-    return matchers
-      .match(pageElement, valueToTextTransformer.transform(value))
-      .then(matcherResult => expect(matcherResult).toBe(true));
-  });
+Then(/^there (is|is no) element "([^"]*)" with value "([^"]*)"$/, function(assert, elementName, value) {
+  return methods.checkers.checkElementWithValue(this.currentPage, assert, elementName, value);
 });
 
-Then(/^there is no element "([^"]*)" with value "([^"]*)"$/, function(elementName, value) {
-  const pageElement = this.currentPage.getElement(elementName);
-
-  return matchers
-    .match(pageElement, valueToTextTransformer.transform(value))
-    .catch(() => Promise.resolve(false))
-    .then(result => (result ? Promise.reject() : Promise.resolve()));
+Then(/^there (is|is no) element "([^"]*)" containing "([^"]*)" text$/, function(assert, elementName, value) {
```

Comment:
Did you checked if autosuggestion works for this step?

Reply:
yes. It gives you a suggestion template `And there is|is no element "<string>" containing "<string>" text`